src/main/webapp/i18n/en/statistics.json

```diff
@@ -12,11 +12,11 @@
         "course_statistics_title": "Course statistics",
         "submissions": "Submissions",
         "active_users": "Active users",
-        "logged_in_users": "Logged in users",
+        "logged_in_users": "Logged-in users",
         "released_exercises": "Released exercises",
         "exercises_due": "Exercises due",
         "conducted_exams": "Conducted exams",
-        "exam_participations": "Exam participations",
+        "exam_participations": "Exam participants",
```

Comment:
This could stay as is, otherwise I'd rename the variable.

Reply:
I disagree purely on the basis that "participation" is not quantifiable; it's a singular-only noun. I think the variable should be renamed, but that was outside the scope of this PR.